website/docs/guides/sl-partner-integration-guide.md

```diff
@@ -142,6 +148,7 @@ These are recommendations on how to evolve a Semantic Layer integration and not
 * Listing available dimensions based on one or many metrics
 * Querying defined metric values on their own or grouping by available dimensions
 * Display metadata from [Discovery API](/docs/dbt-cloud-apis/discovery-api) and other context
+* Use [saved queries](/docs/build/saved-queries) to query pre-build metrics, dimensions, and filters. Refer to the [JDBC](/docs/dbt-cloud-apis/sl-jdbc) and [GraphQL](/docs/dbt-cloud-apis/sl-graphql) APIs for syntax.
```

Comment:
@rpourzand also added this here as integration recommendations - do you think it makes sense?

Reply:
Yeah looks great! I might tweak it slightly to read like 

"Expose "Saved Queries" which are pre-built metrics, dimensions, and filters that Semantic Layer developers create for easier analysis. You can expose them in your application - refer to the JDBC and GraphQL APIs..."